packages/react-lowcode/src/codegen/generation/context/context.ts

```diff
@@ -1,10 +1,12 @@
-import { TableType, UiFramework, Formatter } from '../definition/context-types'
+import { TableType, UiFramework, Formatter } from '../../definition/context-types'
+import { Naming } from '../interfaces/naming';
 
-export default interface GenerationContext {
+export default interface CodegenOptions {
     formatter?: Formatter,
     uiFramework?: UiFramework,
     detail?: DetailGenerationContext,
     index?: IndexGenerationContex
```

Comment:
index is old name for list

I think that options should be serializable (therefore context members are not good for options)